### PR TITLE
fix: Attributes in timeouts are erroneously reversed

### PR DESCRIPTION
**Pull request by qingvincentyin**
## Description
The 2 lines in my proposed change were erroneously reversed in your original code.

### CODE REVIEW

**Comment by bryantbiggs** [comment]
is this change necessary?

**Comment by qingvincentyin** [comment]
Are you asking me? If so, why not? The original code obviously had the 2 attributes transposed.

**Comment by bryantbiggs** [comment]
ooooh, I see it now. Sorry, the information provided in the PR was not clear

File: main.tf [hunk 1]
```diff
@@ -43,8 +43,8 @@ resource "aws_eks_cluster" "this" {
 
   timeouts {
     create = lookup(var.cluster_timeouts, "create", null)
-    delete = lookup(var.cluster_timeouts, "update", null)
```
Comment (bryantbiggs) [inline main.tf:46]: minor nit-pick - could we do the order of `create`, `update`, `delete` (CRUD)

**Review by bryantbiggs — APPROVED** [review]
@antonbabenko should be go with or without the order change

thanks for the catch and fix @qingvincentyin

**Comment by antonbabenko** [comment]
I agree that it should be in order like CRUD. Merging now.

**Comment by antonbabenko** [comment]
This PR is included in version 18.2.2 :tada:

**Pull request merged**